src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  .card__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url('./images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 .card__star:nth-child(-n + 4) {
+    background-image: url('./images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
```

Comment:
styles related to stars itself should be applied for stars BEM block, styles related to stars in card (its external geometry) for card__stars element

Reply:
resolved